src/robusta/integrations/msteams/msteams_elements/msteams_column.py

```diff
@@ -18,14 +18,14 @@ def __column_set(self) -> map:
     def add_column(
         self,
         items: List[MsTeamsBase],
-        width_strech: bool = False,
+        width_stretch: bool = False,
```

Comment:
Since I'm already renaming the file, the fact that I'm changing a named argument is relatively harmless.